av/video/format.pyx

```diff
@@ -128,7 +128,7 @@ cdef class VideoFormatComponent(object):
     property bits:
         """Number of bits in the component."""
         def __get__(self):
-            return self.ptr.depth_minus1 + 1
+            return self.ptr.depth
```

Comment:
Ahaha what a change.

Reply:
I'm really good at math huh? ;)

Reply:
Ah I forgot to remove the comment referring to a bitfield, these are just plain ints